solutions/security/endpoint-response-actions.md

```diff
@@ -165,7 +165,7 @@ Required privilege (in {{stack}}) or custom role privilege (in {{serverless-shor
 Example: `get-file --path "/full/path/to/file.txt" --comment "Possible malware"`
 
 ::::{note}
-The maximum file size that can be retrieved using `get-file` is `104857600` bytes, or 100 MB.
+{applies_to}`stack: removed 9.3`{applies_to}`serverless: removed` The maximum file size that can be retrieved using `get-file` is `104857600` bytes, or 100 MB.
```

Comment:
Since this is an admonition, you should use the [applies_to admonition](https://docs-v3-preview.elastic.dev/elastic/docs-builder/tree/main/syntax/admonitions#applies-to-information) here instead. All you have to do is move the text to the next line and update the applies to formatting a little

```suggestion
:applies_to: {"stack": "removed 9.3", "serverless": "removed"}
The maximum file size that can be retrieved using `get-file` is `104857600` bytes, or 100 MB.
```